doc/Development_Documentation/18_Tools_and_Features/13_Perspectives.md

```diff
@@ -33,29 +35,34 @@ You can restrict the access to certain perspectives via the user settings.
 
 ## The Configuration File
 
-Please refer to the [Example File](https://github.com/pimcore/skeleton/blob/11.x/config/pimcore/perspectives.example.php) 
+Where the perspective configure is stored depends on your configuration.
+Per default this will be the settings-store. 
+If you want to change the default behaviour have a look at the [environment](../21_Deployment/03_Configuration_Environments.md#page_Configuration-Storage-Locations-Fallbacks) configuration options. 
```

Comment:
```suggestion
Per default, this will be the settings-store. 
If you want to change the default behaviour, have a look at the [environment](../21_Deployment/03_Configuration_Environments.md#page_Configuration-Storage-Locations-Fallbacks) configuration options. 
```